assets/css/js_interop.css

```
@@ -319,12 +319,82 @@ solely client-side operations.
   @apply text-green-bright-400;
 }
 
-[data-el-session]:is([data-js-view="code-zen"], [data-js-view="presentation"])
+[data-el-session]:is(
+    [data-js-view="code-zen"],
+    [data-js-view="presentation"],
+    [data-js-view="custom"]
+  )
   [data-el-views-disabled] {
   @apply hidden;
 }
 
-[data-el-session]:not([data-js-view="code-zen"], [data-js-view="presentation"])
+[data-el-session]:not(
+    [data-js-view="code-zen"],
+    [data-js-view="presentation"],
+    [data-js-view="custom"]
+  )
   [data-el-views-enabled] {
   @apply hidden;
 }
+
+/* Custom view */
+
+/* outputs */
+[data-js-view="custom"]:not([data-js-output="true"]) [data-el-output] {
```

I am thinking it will make more sense to do it the opposite. Instead of `data-js-view`+`data-js-output` you could have:

```suggestion
[data-js-hide-output] [data-el-output] {
```

This way, you don't need to check for both data-js-view and data-js-output, only one attribute. And it will make it easier to rewrite both presentation/code-zen using the new hide attributes.